### PR TITLE
feat: implement signer server

### PR DESCRIPTION
**Pull request by aleacevedo**
## Summary

implementation of a Signer Server to be able to make iOS Wallet Integration work.

### CODE REVIEW

File: src/constants/walletconnect.ts [hunk 1]
```diff
@@ -80,3 +80,5 @@ export const SIGNING_METHODS = [
   'requestCall',
   'verifyWhitelist',
 ];
+
+export const SIGNER_SERVER_URL = 'https://signer-server.0xaace.xyz';
```
Comment (mattgle) [inline src/constants/walletconnect.ts:84]: add todo to change later

File: src/redux/slices/walletconnect.js [hunk 2]
```diff
@@ -63,6 +63,7 @@ export const walletConnectOnSessionRequest = createAsyncThunk(
   'walletconnect/onSessionRequest',
   async ({ uri, requestId }, { dispatch, getState }) => {
     try {
+      console.log('URI', uri);
```
Comment (mattgle) [inline src/redux/slices/walletconnect.js:66]: delete log

File: src/services/SignerServer.ts [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  const response = await fetch(`${SIGNER_SERVER_URL}/batchTransaction`, {
+    method: 'POST',
+    body: JSON.stringify({ batchTxData, metadata, host }),
+    mode: 'cors',
+    headers: {
+      'Content-Type': 'application/json',
+      'Access-Control-Allow-Origin': '*',
+    },
+  });
+  console.log('response', response);
```
Comment (mattgle) [inline src/services/SignerServer.ts:18]: delete log

File: src/utils/deepLink.js [hunk 4]
```diff
@@ -33,6 +49,7 @@ function handleWalletConnect(uri, isUnlocked) {
 }
 
 export const handleDeepLink = (url, isUnlocked) => {
+  console.log('handleDeepLink', url);
```
Comment (mattgle) [inline src/utils/deepLink.js:52]: delete

File: ios/Plug/Plug.entitlements [hunk 5]
```diff
@@ -5,6 +5,7 @@
 	<key>com.apple.developer.associated-domains</key>
 	<array>
 		<string>applinks:plugwallet.ooo</string>
+		<string>applinks:qrrgb-uqaaa-aaaad-qd6lq-cai.ic.fleek.co</string>
```
Comment (ignaciosantise) [inline ios/Plug/Plug.entitlements:8]: do we need this url?

File: src/constants/walletconnect.ts [hunk 6]
```diff
@@ -80,3 +80,5 @@ export const SIGNING_METHODS = [
   'requestCall',
   'verifyWhitelist',
 ];
+
+export const SIGNER_SERVER_URL = 'https://signer-server.0xaace.xyz'; //TODO: change to production url
```
Comment (ignaciosantise) [inline src/constants/walletconnect.ts:84]: move to constants/urls

File: ios/Plug.xcodeproj/project.pbxproj [hunk 7]
```diff
@@ -581,6 +582,7 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CLANG_ENABLE_MODULES = YES;
+				CODE_SIGN_ENTITLEMENTS = Plug/Plug.entitlements;
```
Comment (aleacevedo) [inline ios/Plug.xcodeproj/project.pbxproj:585]: @ignaciosantise do u know why this change?

**Review by mattgle — APPROVED** [review]
